cpp/src/parquet/column_writer.cc

```diff
@@ -1104,6 +1106,7 @@ int64_t ColumnWriterImpl::Close() {
     if (rows_written_ > 0 && chunk_statistics.is_set()) {
       metadata_->SetStatistics(chunk_statistics);
     }
+    metadata_->SetKeyValueMetadata(std::move(key_value_metadata_));
```

Comment:
🤔 should for not moved in to help some debugging?

Reply:
done